fixtures/small/heredoc_param_actual.rb

```diff
@@ -0,0 +1,22 @@
+puts <<~STRING, <<-STRING2,
+    yellow
+    fruit
+    peel
+  STRING
+    cheese
+    noodle
+    sauce
+  STRING2
+  :burrito,
+  <<~STRING3,
+    jasmine
+    earl grey
+    oolong
+  STRING3
+  :rice
+
+puts [:one, <<~TWO, :three], :four, [:five, <<~SIX, :seven], :eight
+two
+TWO
+six
+SIX
```

Comment:
:thought_balloon: This part of the test did not expose any additional problems with trunk-rubyfmt, but it did expose a problem with an earlier iteration of this PR so I left it in.